src/helpers/geoChart/data.ts

```diff
@@ -129,9 +129,10 @@ function getBucketItemNameAndDataIndex(
                     attributeHeader.uri === bucketItemInfo.uri,
             );
             if (index !== -1) {
+                const { formOf: { name = attributeHeaders[index].name } = {} } = attributeHeaders[index];
```

Comment:
Please make a variable for `attributeHeaders[index]` :)

Reply:
Currently, the `formOf` always has in `attributeHeader` so I changed the code to:
```
const { formOf: { name } } = attributeHeaders[index];
result[bucketName] = { index, name };
```